exporter/stackdriver/examples/stats/main.go

```diff
@@ -1,82 +0,0 @@
-// Copyright 2017, OpenCensus Authors
```

Comment:
I have a question. Is it possible to keep this package with a doc.go and say it has moved to the new location for now?

Then it will break the builds and people will read the godoc and migrate to the new import path. Then, in a few releases, we can delete the package entirely.

Reply:
Yes, I will do that.